Sources/PointFree/Collections/CollectionIndex.swift

```diff
@@ -14,13 +14,14 @@ let collectionsIndexMiddleware: M<Tuple3<User?, SubscriberState, Route?>>
     <| map(lower)
     >>> writeStatus(.ok)
     >=> respond(
-      view: { collectionIndex },
+      view: collectionIndex(collections:),
       layoutData: { currentUser, subscriberState, route in
         SimplePageLayoutData(
           currentRoute: route,
           currentSubscriberState: subscriberState,
           currentUser: currentUser,
-          data: (),
+          data: Current.collections,
+          extraStyles: collectionIndexStyles,
```

Comment:
a few extra styles we have to inject into this view, but as I'll explain in a bit I think we could even get rid of this later...